PokeOneWeb/Frontend/pokeoneweb/src/app/core/services/api/location.service.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { ILocationListModel } from '../../models/location-list.model';
+import { BaseService } from './base.service';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class LocationService extends BaseService {
+    constructor(http: HttpClient) {
+        super(http);
+    }
+
+    public getPathSegment(): string {
+        return 'locationGroups';
+    }
+
+    public getAllForRegion(regionName: string): Observable<ILocationListModel[]> {
+        return this.http.get<ILocationListModel[]>(
+            `${environment.baseUrl}/regions/${regionName}.json`,

```

Comment:
Do you get the right data from these branches? Looks like they only contain information about the region, but not its location groups...

Reply:
yes, the page for the location groups is action the region-page -> for every region of the dropdown there is one json file with all location groups in this region. Otherwise the current JSON-Files would not make any sense. Because the folder location-groups contains 1 JSON file for every locaiton group